radio/src/pulses/pxx_arm.cpp

```diff
@@ -307,6 +311,13 @@ void setupPulsesPXX(uint8_t port)
   /* PPM */
   static uint8_t pass[NUM_MODULES] = { MODULES_INIT(0) };
   int sendUpperChannels = 0;
+  static unsigned int iError[2][MAX_OUTPUT_CHANNELS];
+  static int lastPPM[2][MAX_OUTPUT_CHANNELS];
+  for (int i = 0; i < MAX_OUTPUT_CHANNELS; ++i)
+  {
+      iError[port][i] += abs( (short)((channelOutputs[i]) -lastPPM[port][i]) );
+      iError[port][i] =MAX_IERROR>iError[port][i]? iError[port][i]: MAX_IERROR;
+  }
   if (pass[port]++ & 0x01) {
```

Comment:
this code is now only needed for failsafe frames

Reply:
this code just for judge 1--8 and 9--16 channel which has big change.